src/sql/base/browser/ui/chart/doughnutChart/doughnutChart.component.html

```diff
@@ -0,0 +1,11 @@
+<div style="display: block; width: 100%; height: 100%; position: relative;">
+	<canvas baseChart tabindex="0"
```

Comment:
So none of these seem to be specifically for donut chart. My overall suggestion is that instead of making just a donut chart make a generic "chart" component that extensions can make and pass data to, which includes the chart type (donut, bar, etc.)

So basically this PR would have almost nothing specific to a donut chart - it's only in the extension that the user would specify `chartType: 'donut'` to display the chart as a donut. This would even allow cool stuff like being able to have a button that switches the chart type, so users can toggle between different types without needing to recreate the entire chart component each time!

Reply:
created a generic chart component